CustomCraftSML/SMLHelperItems/CustomFabricatorBuildable.cs

```diff
@@ -23,7 +23,7 @@ public CustomFabricatorBuildable(CustomFabricator customFabricator)
         public override TechGroup GroupForPDA => FabricatorDetails.PdaGroup;
         public override TechCategory CategoryForPDA => FabricatorDetails.PdaCategory;
         public override string AssetsFolder { get; } = FileLocations.RootModName + "/Assets";
-        public override string IconFileName => $"{FabricatorDetails.Model}.png";
+        public override string IconFileName => FabricatorDetails.ItemImage;
```

Comment:
Additional TODO: Update `CustomFabricatorBuildable` to inherit from `ModPrefab` instead of `Buildable`.  
Then, handle all patching manually, using a combination of the patching methods provided by `AliasRecipe` and any new patching methods needed.  
This will be the pattern going forward for all elements in the `SMLHelperItems` folder.

Also, correct the namespace to match the project structure.